### ISSUE TITLE
SRIGQter Cannot Import Splunk STIG v1r1

### ISSUE THREAD

**Issue by TheCleverEpithet** [squinky86/STIGQter#86]
STIGQter v1.0.1 does not seem to be able to import the new Splunk STIG found at the link below.  

https://dl.dod.cyber.mil/wp-content/uploads/stigs/zip/U_Splunk_Enterprise_7-x_for_Windows_V1R1_STIG.zip

It gives no errors, and acts like it imports, but nothing is listed for it in the STIG list pane and if you try to import a Checklist file created from this STIG it does give an error that the natch STIG has not been imported.  Not clear if this is an issue with the STIG from DISA or something in STIGQter.

**Comment by TheCleverEpithet** [comment]
Problem was possibly identified.  This was a statement from the user who initially reported the issue and found a workaround:

_"I think I may have found out why it will not import. Unlike every other STIG (at least the many that I checked) the Splunk STIG ends with STIG.xml. All others end with xccdf.xml. I changed the file name and it appears to be working. So it seems STIGQter is looking for a file with xccdf in the name. I’m testing it now, I’ll let you know the result." [...] "Yes, it worked."_
